.github/workflows/push-1password-secrets-to-gha.yaml

```diff
@@ -8,14 +8,15 @@ on:
   push:
     branches:
       - "push-secrets-*"
+  workflow_dispatch:
 
 concurrency:
   group: ${{ github.workflow }}
   cancel-in-progress: false
 
 jobs:
-  push-secrets:
-    name: "Push Secret"
+  push-github-secrets:
+    name: "Push Github Secret"
```

Comment:
should we also rename `sync-secrets` -> `sync-secrets-gh`?

Reply:
yes!

Reply:
pushed!